src/macros.yay

```diff
@@ -112,7 +113,7 @@ $(macro) {
 
 $(macro) {
     compile {
-        function $(llvmidentifier() as name) ($(optional(ls(
+        function $(llvmidentifier() as name) $! ($(optional(ls(
```

Comment:
This seems to be breaking function calls with no arguments

```
compile {
     foo($x);
}
```

works

```
compile {
     foo();
}
```

gives me a fatal error/throws a runtime exception in the docker image.

Reply:
That line shouldn't trigger that error, since there is no function keyword... It would trigger the other instance though...

Fixed, with test added.

Reply:
you're right, I commented on the wrong line